include/swift/SIL/SILInstruction.h

```diff
@@ -3914,7 +4238,8 @@ class ReturnInst
 /// ThrowInst - Throw a typed error (which, in our system, is
 /// essentially just a funny kind of return).
 class ThrowInst
-  : public UnaryInstructionBase<ValueKind::ThrowInst, TermInst,
+  : public UnaryInstructionBase<ValueKind::ThrowInst,
+                                TermInst,
```

Comment:
Instead of finding more synonyms for "ditto", I'll just ask you to look for all the other examples of this. :)